mods/persistence/modules/overmap/sectors.dm

```diff
@@ -3,7 +3,7 @@
 					  		 // If the area or z-level is saved, the overmap effect will be saved.
 	var/atom/old_loc	 	 // Where the ship was prior to saving. Used to relocate the ship following saving, not on load.
 
-	//#FIXME: Don't store this here.
+	//#FIXME: Don't store this here. Those are meant to be strictly just temporary objects just to show a position on the overmap!
```

Comment:
I don't think these should be regarded as temporary, since ship velocity etc. are stored on this object.

Reply:
The thing is, that's what neb has been going for last time I asked. That's part of why most of the planet code was decoupled from the markers.

Reply:
Not blocking, but it's not like Nebula has saving either. There's nothing in game that creates and destroys these objects, so I'm not sure how they could be regarded as temporary.